.gitlab-ci.yml

```diff
@@ -1009,6 +1009,15 @@ stages:
     JANE_DIFF_PATTERN: '.*/debops.roundcube/.*'
     JANE_LOG_PATTERN: '\[debops\.roundcube\]'
 
+'roundcube/mariadb role':
+  <<: *test_role_2rd_deps
+  variables:
+    JANE_TEST_PLAY: '${DEBOPS_PLAYBOOKS}/service/mariadb_server.yml ${DEBOPS_PLAYBOOKS}/service/roundcube.yml'
+    JANE_INVENTORY_GROUPS: 'debops_service_mariadb_server,debops_service_roundcube'
+    JANE_INVENTORY_HOSTVARS: 'roundcube__database=mysql-default'
```

Comment:
The role has no database autodetection implemented? That's fine, we can do it later. The goal should be that extra variables for tests aren't needed, that way the roles can automatically adapt to the existing environment.

Reply:
The role still uses the ancient database configuration table lookup pattern (originally copied from the wordpress role, or the like). IMHO, SQLite is a much better choice for most installations as the database has only infrequent updates and a file is much easier to backup. A real DBMS is only needed if you have multiple Roundcube instances and then you would want a separate database server anyway.

If you feel MariaDB should be auto-configurable I can send a PR for this after v0.7.0. It likely would break existing installations that's another reason why I hesitated to change this so far. But it might be a good occasion now, as there is some compatibility code which could be cleaned up in the role anyway.

Reply:
Sure, doing that after `v0.7.0` is released is fine. I want to add PostgreSQL support as well, so we will se how that goes. Another thing to consider is `memcached` autodetection to switch session management from database to memcache.